Analysis/config/2017/sources.cfg

```diff
@@ -1,32 +1,43 @@
 [ANA_DESC common]
 int_lumi: 41529.343
 period: Run2017
-apply_mass_cut: true
+final_variables: m_ttbb_kinfit MT2 mva_score
 energy_scales: all
-categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF 2j1b+B 2j1b+B_noVBF 2j0b 2j1b 2j2b+ 2j0Lb 2j1Lb 2j2Lb
-sub_categories: mh
+#categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF
+#categories: 2j 2j0bR_noVBF 2j1bR_noVBF 2j2b+R_noVBF 2j2Lb+B_noVBF 4j1b+_VBF 2j1b+B 2j1b+B_noVBF 2j0b 2j1b 2j2b+ 2j0Lb 2j1Lb 2j2Lb 2j1bR 2j2b+R
+categories: 2j
+sub_categories: NoCuts
+#sub_categories: mh
 regions: OS_Isolated OS_AntiIsolated SS_Isolated SS_AntiIsolated SS_LooseIsolated
-limit_setup: MVA_res mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF m_ttbb_kinfit:4j1b+_VBF
+limit_setup: MVA_res mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF  m_ttbb_kinfit:4j1b+_VBF
 limit_setup: MVA_nonres mva_score:2j1bR_noVBF,2j2b+R_noVBF,2j2Lb+B_noVBF MT2:4j1b+_VBF
+limit_category: 2j1bR_noVBF res1b_noVBF
```

Comment:
`limit_category` and `limit_categories` should be removed from `sources.cfg` and `AnalyzerSetup`.